lld/test/ELF/invalid/symtab-symbols.test

```diff
@@ -1,3 +1,6 @@
+## Temporary disabled for Windows, fix is required for ld.lld non-zero exit code
```

Comment:
Similar query to that above for lld/test/ELF/eh-frame-type.test.  Likewise, I suspect from the names, for the following few files.